### PR TITLE
Use `-cl-fp32-correctly-rounded-divide-sqrt` for LTS

### LINKED ISSUE #6992: [Triton 3.7.1] Inductor inference latency regression on PVC — 50 models affected (10–20x slower)
## Summary

Triton 3.7.1 introduces severe inductor latency regressions on **PVC (Ponte Vecchio)** for **inference-only** workloads. 50 model/dtype/mode combinations are affected, with inductor latency increasing **10–20x** compared to Triton 3.7.0 baseline.

**Training workloads are unaffected.** Eager latency is stable between target and baseline, confirming this is a Triton compilation/codegen issue, not an environment problem.

## Environment

| Component | Target | Baseline |
|-----------|--------|----------|
| Triton | 3.7.1 (`2bafaf80`) | 3.7.1 (`21033c4e2be`) |
| PyTorch | torch 2.12.0+xpu (nightly) | torch 2.12.0+xpu (nightly) |
| Platform | PVC (Data Center Max) | PVC (Data Center Max) |

## Affected Models

All are **inference mode** across multiple dtypes (amp_bf16, amp_fp16, bfloat16, float16).
Ratio = baseline_latency / target_latency (values < 1.0 mean target is slower):

| Model | Suite | Inductor Ratio | Eager Ratio |
|-------|-------|----------------|-------------|
| fbnetc_100 | timm_models | **0.054x** | 1.00x |
| ghostnet_100 | timm_models | **0.062x** | 1.00x |
| mnasnet_100 | timm_models | **0.055x** | 1.00x |
| gluon_inception_v3 | timm_models | **0.066x** | 1.03x |
| inception_v3 | timm_models | **0.065x** | 1.03x |
| res2net50_14w_8s | timm_models | **0.063x** | 1.00x |
| resnest101e | timm_models | **0.065x** | 1.00x |
| timm_resnest | torchbench | **0.065x** | 1.01x |
| botnet26t_256 | timm_models | **0.062x** | 1.00x |
| resnet152 | torchbench | **0.069x** | 1.00x |
| resnet50 | torchbench | **0.063x** | 1.00x |
| resnext50_32x4d | torchbench | **0.069x** | 1.00x |
| densenet121 | torchbench | **0.079x** | 1.00x |
| mobilenet_v2 | torchbench | **0.051x** | 1.00x |
| mobilenetv2_100 | timm_models | **0.050x** | 1.00x |
| mnasnet1_0 | torchbench | **0.057x** | 1.00x |
| spnasnet_100 | timm_models | **0.055x** | 1.00x |

## Verification

- **Eager latency ratio is ~1.0x** for all affected models, confirming the test environment is consistent.
- **Training mode is unaffected** — same models in training show inductor latency ratio within 0.97–1.03x.
- Results reproduced on rerun.

## Observations

- All affected architectures use **depthwise separable convolutions** or **mobile-style inverted residual blocks** (MobileNet family, EfficientNet family, ResNeSt, GhostNet).
- The regression appears to be in Triton kernel codegen or scheduling for these specific graph patterns during inference.
- The 10-20x regression suggests compilation is falling back to a suboptimal code path or generating excessive kernel launches.

## Steps to Reproduce

```bash
docker run -it -e TZ=Aisa/Shanghai --device=/dev/mem --device=/dev/dri --group-add video \
    --privileged --shm-size=8g intelgpu/ubuntu-24.04-lts2:2523.40 bash

# setup python
curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/usr/local/bin" sh
uv venv myvenv --python 3.12 --clear
source myvenv/bin/activate
uv pip install pip numpy 'setuptools<81' wheel

# install pytorch nightly xpu wheel
uv pip install torch torchaudio torchvision --pre --index-url https://download.pytorch.org/whl/xpu

# prepare pytorch source
pytorch_commit="$(python -c 'import torch; print(torch.version.git_version)')"
git clone https://github.com/pytorch/pytorch
cd pytorch
git checkout ${pytorch_commit}

# install deps
uv pip install pip pandas psutil scipy requests
uv pip install -U numpy==1.26.4 accelerate
uv pip install -r .ci/docker/ci_commit_pins/huggingface-requirements.txt
uv pip install timm

# sync model list
rm -rf torch torch-xpu-ops
git clone https://github.com/intel/torch-xpu-ops
rsync -avz ./torch-xpu-ops/.ci/benchmarks/ ./benchmarks/dynamo/

# optional for torchbench models only
git clone https://github.com/pytorch/benchmark tmp-benchmark
cd tmp-benchmark
python install.py --continue_on_fail <modela,modelb,...>
cd ..

# uninstall the triton and install new from https://github.com/pytorch/pytorch/actions/runs/25853155119?pr=182438

# run affected models (inference)
python benchmarks/dynamo/torchbench.py -d xpu --performance --bfloat16 \
  --inference --backend=inductor --cold-start-latency -n10 \
  --disable-cudagraphs --only resnet50

python benchmarks/dynamo/timm_models.py -d xpu --performance --bfloat16 \
  --inference --backend=inductor --cold-start-latency -n10 \
  --disable-cudagraphs --only mobilenetv2_100
```

## Workflow Runs

- Target: https://github.com/intel/torch-xpu-ops/actions/runs/25921168232
- Baseline: https://github.com/intel/torch-xpu-ops/actions/runs/25737207406

## Expected Behavior

Inductor latency should be comparable to or better than baseline for inference workloads (ratio ~1.0x).

### PR DESCRIPTION
**Pull request by anmyachev**
Relates to #6992

https://github.com/intel/intel-xpu-backend-for-triton/issues/6261 works with these changes (tested original E2E model)

Inductor: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/26450746158 (to check)

### CODE REVIEW

**Review by whitneywhtsang — APPROVED** [review]

**Comment by whitneywhtsang** [comment]
Should we do this change in main branch first?

**Comment by anmyachev** [comment]
> Should we do this change in main branch first?


Maybe we should do it the other way around this time? Cherry-pick the commit from the release branch to the main branch? It'll be faster, since we're currently dealing with issues with GitHub, so it could take a while.

**Comment by whitneywhtsang** [comment]
> > Should we do this change in main branch first?
> 
> Maybe we should do it the other way around this time? Cherry-pick the commit from the release branch to the main branch? It'll be faster, since we're currently dealing with issues with GitHub, so it could take a while.

That's fine with me.

**Review by dev-tomek — APPROVED** [review]
